### ISSUE TITLE
BB4_P7 (Book 4 p. 12)

### ISSUE THREAD

**Issue by rryman** [rryman/BB4-Worksheets#20]
Suggested rewording:

Copy each “x” at the correct placement in the empty stage plan below, then copy the
corresponding location sign directly below the stage plan. Pay special attention to the horizontal
drawing of the tick: its placement indicating how far upstage or downstage, and its positioning “on the location,” “right of location,” “left of location,” or “between” locations.

**Comment by rsryman** [comment]
This issue refers to Worksheets p. 7. 
There are actually 3 elements. This is a bit wordy, but there is room. For sentence 2, what about:

Pay special attention to the horizontal drawing of the tick; to its placement along the sign (indicating how far the dancer is upstage or downstage); and to its positioning centred on, at one side of, or between the sign/signs (indicating “on the location,” “right of location,” “left of location,” or “between locations").

**Comment by rryman** [comment]
That works. Once you have implemented it, please close this comment.

**Comment by rsryman** [comment]
Done.
